src/interpreter/executor/interpret.js

```diff
@@ -262,6 +288,22 @@ function evalState(buffer) {
 
         case stateEnum.ASSIGNING_VAR:
             break;
+
+        case stateEnum.ACCEPTING_CONSOLE:
+            buffer = acceptingConsole(buffer);
+            break;
+    }
+    return buffer;
+}
+
+// If we are taking in console input
+function acceptingConsole(buffer) {
+    
+    //There has to be a better way
+    if (buffer[buffer.length - 1] === ')') {
```

Comment:
I think this can work, but it assumes that the line for a log statement is formatted exactly like this:
`console.log(' <text> ')` , which is not the only way to have valid javascript.

Reply:
What do you mean by this? wouldn't console log always end with a paren and open with one? I grab both indexes and substring on it.